spec/libraries/has_many_inverse_through_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe 'Extensions: has_many inverse_through', type: :model do
+  temporary_table(:stores) do |t|
+  end
+
+  temporary_table(:products) do |t|
+    t.references :store
+    t.references :product, polymorphic: true
+  end
+
+  temporary_table(:pens) do |t|
```

Comment:
Unused block argument - `t`. You can omit the argument if you don't care about it.